src/content/development/contribution-guide/_index.en.md

```diff
@@ -56,9 +56,41 @@ To be expanded
 
 To be expanded
 
-#### Pre-Planning
+#### Planning
 
-To be expanded
+The week before a new release cycle starts is dedicated to planning the release.
+Planning accounts for [epics], [tasks] and [bugs] which we intend to handle in the next version.
```

Comment:
Twitching a bit at the lack of serial comma but it's fine. It doesn't add ambiguity here, which is I guess why it might be a blocker.

https://en.wikipedia.org/wiki/Serial_comma#Arguments_for_and_against